partest-suite/src/test/resources/scala/tools/partest/scalajs/2.12.0-M4/BuglistedTests.txt

```diff
@@ -0,0 +1,15 @@
+# The tests in this file should pass but have never passed so far
+# use scala.tools.partest.scalajs.testunknownonly to only run tests
+# which are neither in BuglistedTests.txt, WhitelistedTests.txt or
+# BlacklistedTests.txt
+
+# Bug #2309
+run/sammy_after_implicit_view.scala
+run/sammy_restrictions_LMF.scala
+run/sammy_vararg_cbn.scala
+run/t5568.scala
```

Comment:
https://github.com/scala/scala/blob/2.12.x/test/files/run/t5568.scala
Uh? How is this related to #2309?